tests/lib/collectors/osquery.py

```diff
@@ -87,6 +87,55 @@ def testProcess(self, mock_exists) -> None:
     self.assertEqual(len(containers), 1)
     self.assertEqual(containers[0].query, "SELECT * FROM processes")
 
+  @mock.patch('os.path.exists')
+  def testProcessQueryPack(self, mock_exists) -> None:
+    """Tests the collector's Process() function with a Osquery Pack file."""
+    test_state = state.DFTimewolfState(config.Config)
+    osquery_collector = osquery.OsqueryCollector(test_state)
+    mock_exists.return_value = True
+
+    test_ok_data = """{
+      "platform": "darwin",
+      "queries": {
+        "query_1": {
+          "query": "select * from launchd where path like '%System.plist';",
+          "interval" : "3600",
+          "version": "1.4.5",
+          "description" : "description",
+          "value" : "Artifact used by this malware"
+        },
+        "query_2": {
+          "query" : "select * from test where path like '%user32.dll';",
+          "interval" : "3600",
+          "version": "1.4.5",
+          "platform": "windows",
+          "description" : "description",
+          "value" : "Artifact used by this malware"
+        }
+      }
+    }"""
```

Comment:
Please fix the spaces throughout this whole blob (that's why I suggested using dumps in the first place heh)

Reply:
My bad, thought I updated but guess the update didn't get pushed up.  Should be fixed now (pending PR actions).